Source/GRTeclynCore/GRAMRLevel.cpp

```diff
@@ -199,6 +199,19 @@ void GRAMRLevel::post_timestep(int /*iteration*/)
 
         FourthOrderInterpFromFineToCoarse(S_crse, 0, NUM_VARS, S_fine, ratio);
     }
+    if (simParams().nan_check)
+    {
+        amrex::MultiFab &state_new = get_new_data(State_Type);
+        if
+
+            (state_new.contains_nan(0, state_new.nComp(), amrex::IntVect(0),
+                                    true))
+
+        {
+
+            amrex::Abort("NaN in GRAMRLevel::post_timestep");
```

Comment:
I'm a bit surprised clang-format was OK with all of this unnecessary whitespace but let's get rid of it:
```suggestion
        if (state_new.contains_nan(0, state_new.nComp(), amrex::IntVect(0),
                                   true))
        {
            amrex::Abort("NaN in GRAMRLevel::post_timestep");
```

Reply:
I think, I messed up something while rebasing. The PR should be clean now, just faling because of Tursa is down.

Reply:
I think the whitespace is still there. We don't use Tursa for CI but CSD3. I believe it is failing because some of the login nodes are down for maintenance.

Reply:
It's gone now!